api/api/controllers/search_controller.py

```diff
@@ -349,12 +349,14 @@ def search(
         ("extension", None),
         ("category", None),
         ("categories", "category"),
+        ("source", None),
+        ("license", None),
+        ("license_type", "license"),
```

Comment:
Not related to this PR, but the `license_type` should not be the same as `license`... 
I know it has been for a long time, and changing it would probably be a breaking change. We should probably deprecate this parameter.
We're not using this filter in Nuxt, by the way.

Reply:
Yeah changes to `license_type` would be breaking changes, so it's best to not disturb this, for now.